### DISCUSSION TITLE
🔨iLogtail开发者双周会（12月7日 19:30-20:30）

### DISCUSSION THREAD

**Discussion by linrunqi08** [alibaba/ilogtail#528] [Announcements]
时间：12月7日 周三 19:30 - 20:30
会议号：547 589 68085
电话呼入：
+867936168776（中国大陆）
+867936168756（中国大陆）
入会链接：https://meeting.dingtalk.com/j/Ex7D6VU311K

议题
iLogtail近期新功能和修复(zstd/docker fd leak/file fd leak/file重复采集). @yyuuttaaoo 
ilogtail数据流水线v2 支持通用时序数据. @liuhaoyang 
ck flusher开发启动和logtail + clickhouse + clickvisual的方案. @kl7sn

**Comment by kl7sn** [comment]
logtail + clickhouse + clickvisual 方案 https://github.com/alibaba/ilogtail/discussions/554

**Comment by linrunqi08** [comment]
本周双周会视频：https://www.bilibili.com/video/BV1rG4y137Kb/?spm_id_from=333.999.0.0&vd_source=a89cd37d6182e220ca4db38d937a5d94
